conformance/ConformanceApi.fsd

```diff
@@ -121,10 +122,11 @@ service ConformanceApi
 	data Widget
 	{
 		/// A unique identifier for the widget.
+		[validate(value: 0..)]
 		id: int32;
 
 		/// The name of the widget.
-		[required]
+		[required, validate(regex: "[_a-zA-Z0-9]+")]
```

Comment:
```suggestion
		[required, validate(regex: "^[_a-zA-Z0-9]+$")]
```

Reply:
Oh right. 👍

Reply:
Should we just add the anchors implicitly? Is there ever a case where you might want to only match part of a string to determine validity?

Reply:
I'm fine sticking with OpenAPI behavior here, which doesn't have implicit anchors. I suppose one might want to enforce a prefix or a suffix.